stream-chat-android-ui-components/src/main/res/values/styles.xml

```diff
@@ -278,6 +278,7 @@
         <item name="streamUiThreadAvatarStyle">@style/StreamUi.Thread.Avatar</item>
         <item name="streamUiReplyAvatarStyle">@style/StreamUi.Reply.Avatar</item>
         <item name="streamUiMessageOptionsAvatarStyle">@style/StreamUi.MessageOptions.Avatar</item>
+        <item name="streamUiMessageInputViewStyle">@style/StreamUi.MessageInputView</item>
```

Comment:
There is no need to pass default style here because we already did it while obtaining attributes in `MessageInputViewStyle` here:
`    context.obtainStyledAttributes(
                attrs,
                R.styleable.MessageInputView,
                R.attr.streamUiMessageInputViewStyle,
                R.style.StreamUi_MessageInputView,
)`

Reply:
I'm removing this and I'll add Jira task to update docs about theming MessageInputView.